front/components/ConnectorPermissionsModal.tsx

```diff
@@ -24,6 +24,13 @@ const CONNECTOR_TYPE_TO_NAME: Record<ConnectorProvider, string> = {
   github: "GitHub",
 };
 
+const CONNECTOR_TYPE_TO_RESOURCE_NAME: Record<ConnectorProvider, string> = {
+  notion: "top-level Notion pages or databases",
+  google_drive: "Google Drive folders",
```

Comment:
top-level would seem appropriate here too? --> "top-level Google Drive folders"

Reply:
If it's not a requirement anymore by users would be super nice to enforce the selection of shared drives only :)